hazelcast/src/main/java/com/hazelcast/replicatedmap/impl/operation/CheckReplicaVersion.java

```diff
@@ -82,12 +82,15 @@ public void run() throws Exception {
     private void requestDataFromOwner(String name) {
         OperationService operationService = getNodeEngine().getOperationService();
         RequestMapDataOperation requestMapDataOperation = new RequestMapDataOperation(name);
-        operationService.invokeOnPartition(SERVICE_NAME, requestMapDataOperation, getPartitionId());
+        operationService
+                .createInvocationBuilder(SERVICE_NAME, requestMapDataOperation, getPartitionId())
+                .setTryCount(INVOCATION_TRY_COUNT)
+                .invoke();
     }
 
     @Override
-    public boolean returnsResponse() {
-        return false;
+    public Object getResponse() {
```

Comment:
When do we use this response? Relying on defaults in `AbstractOperation` does not help, if it helps no need to override methods `returnsResponse ` and `getResponse `?